### PR TITLE
Add check-cloudformation to precommit/CodeBuild and correct rds-postgres template

### PR DESCRIPTION
**Pull request by WillGibson**
Co-authored-by: @yusufsheiqh

### CODE REVIEW

File: templates/addons/env/rds-postgres.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ Resources:
     Type: AWS::RDS::DBSubnetGroup
     Properties:
       DBSubnetGroupDescription: Group of subnets to place DB into
-      SubnetIds: [ !Ref PrivateSubnet1, !Ref PrivateSubnet2 ]
+      SubnetIds: !Split [ ",", !Ref PrivateSubnets ]
```
Comment (WillGibson) [inline templates/addons/env/rds-postgres.yml:43]: We missed this one the other day @lgarvey 😬

**Review by yusufsheiqh — APPROVED** [review]

File: templates/addons/env/rds-postgres.yml (hunk 1 above)
Comment (lgarvey) [reply templates/addons/env/rds-postgres.yml:43]: Ouch!

Comment (WillGibson) [reply templates/addons/env/rds-postgres.yml:43]: It kind of demonstrates the use case for automated checks 😜

Comment (lgarvey) [reply templates/addons/env/rds-postgres.yml:43]: Why wasn't it picked up by the checks?

Comment (WillGibson) [reply templates/addons/env/rds-postgres.yml:43]: They were not running the pre-commit or the pipeline, this was picked up when I added them in.

**Pull request merged**